torchtune/_recipe_registry.py

```diff
@@ -32,8 +32,8 @@ class Recipe:
         supports_distributed=True,
     ),
     Recipe(
-        name="dev/grpo_torchrl_full_finetune_distributed",
-        file_path="dev/grpo_torchrl_full_finetune_distributed.py",
+        name="dev/async_grpo",
```

Comment:
I think this should be async_distributed_grpo since all our other distributed recipes are explicit about it